src/jltypes.c

```diff
@@ -2069,43 +2069,45 @@ void jl_init_types(void) JL_GC_DISABLED
     jl_code_info_type =
         jl_new_datatype(jl_symbol("CodeInfo"), core,
                         jl_any_type, jl_emptysvec,
-                        jl_perm_symsvec(17,
-                            "code",
-                            "codelocs",
-                            "ssavaluetypes",
-                            "ssaflags",
-                            "method_for_inference_limit_heuristics",
-                            "linetable",
-                            "slotnames",
-                            "slotflags",
-                            "slottypes",
-                            "rettype",
-                            "parent",
-                            "min_world",
-                            "max_world",
-                            "inferred",
-                            "inlineable",
-                            "propagate_inbounds",
-                            "pure"),
-                        jl_svec(17,
-                            jl_array_any_type,
-                            jl_any_type,
-                            jl_any_type,
-                            jl_array_uint8_type,
-                            jl_any_type,
-                            jl_any_type,
-                            jl_array_symbol_type,
-                            jl_array_uint8_type,
-                            jl_any_type,
-                            jl_any_type,
-                            jl_any_type,
-                            jl_long_type,
-                            jl_long_type,
-                            jl_bool_type,
-                            jl_bool_type,
-                            jl_bool_type,
-                            jl_bool_type),
-                        0, 1, 17);
+                        jl_perm_symsvec(18,
+                                        "code",
```

Comment:
code editor seems to be auto-reindenting this block unnecessarily? The intent of the current alignment is to prevent these sorts of excess diff noise, so that adding removing fields and or changing variable names doesn't require excess conflict regions and an indent-aware find-replace editor.

Reply:
should be fixed